bucket/slash32.go

```diff
@@ -60,8 +79,8 @@ func (b *Slash32) Register(r *http.Request, cost float64) {
 	}
 
 	ipnet := "0.0.0.0/0"
-	var network *net.IPNet
-	_, network, err = net.ParseCIDR(fmt.Sprintf("%s/%d", ip, b.netmask))
+	_, network, err := net.ParseCIDR(fmt.Sprintf("%s/%d", ip, b.netmask))
+	// @todo shouldn't this error be handled properly? is it ipv6 compat?
```

Comment:
While reading through this, what happens if there is ipv6? not supported or will it just work?